pages/common/iperf3.md

```diff
@@ -0,0 +1,23 @@
+# iperf3
+
+> Iperf3 — traffic generator for testing network bandwidth
+
+- Run iperf3 as a server
+
+`iperf3 -s`
+
+- Run iperf3 server on port 5209
```

Comment:
Perhaps tokenize the port. Such as `Run an iperf3 server on a specific port:` and then
`iperf3 -s -p{{port}}`